Http.cs

```diff
@@ -139,11 +125,59 @@ public static HttpRequest Post(string uri, Dictionary<string, string> formData)
 			return new HttpRequest(UnityWebRequest.Post(uri, formData));
 		}
 
+		/// <summary>
+		/// Create a HttpRequest configured to send form multipart form to a server via HTTP POST.
```

Comment:
I believe function descriptions should be in 3rd person

Reply:
As in `Creates` over `Create`.

The former will read like `(it) Creates a HttpRequest`. 

The later is less explicit. `Create a HttpRequest` could be either:
1. Imperative like a command `Create a HttpRequest, at once!`
2. Describing what you could do, `(You can) Create a HttpRequest` even though it's not you performing the action it's the program. In which case it suggests it should be 3rd person
3. Describing the action of the program in future tense, (since execution will be in the future from now) 
`(It will) Create a HttpRequest`